src/Sql/Sql/Properties/Resources.resx

```diff
@@ -580,4 +580,16 @@
   <data name="VulnerabilityAssessmentOperationIsNotAvailableOnMasterDatabase" xml:space="preserve">
     <value>Vulnerability Assessment operation is not available on master database. Skipping...</value>
   </data>
+  <data name="DoYouWantToProceed" xml:space="preserve">
+    <value>Do you want to proceed?</value>
+  </data>
+  <data name="ChangingHardwareFamilyIsIrreversable" xml:space="preserve">
+    <value>For this update operation provided compute generation parameter is Gen5. Once the hardware generation is upgraded, it is not possible to switch back to Gen4 compute generation as this hardware is being deprecated.</value>
+  </data>
+  <data name="CannotChangeHardwareFamily" xml:space="preserve">
+    <value>You cannot change hardware family.</value>
+  </data>
+  <data name="InvalidManagedInstanceSkuName" xml:space="preserve">
+    <value>Provided SkuName '{0}' is invalid '{1}'. Please enter a valid SkuName value and execute command again.</value>
```

Comment:
It would be nice to give example sku name so that customer has an better idea of what we're asking for

Reply:
This message is no longer needed. Removing it.